scripts/capacities.py

```diff
@@ -45,45 +29,45 @@ def potentials(path_to_units, path_to_eez, path_to_shared_coast,
     * allocate the offshore potentials to exclusive economic zones (EEZ),
     * allocate the offshore potential of EEZ to units based on the fraction of shared coast.
     """
-    with rasterio.open(path_to_eligibility_categories, "r") as src:
+    with rasterio.open(str(path_to_eligibility_categories), "r") as src:
```

Comment:
I don't like this too much because this is a Snakemake detail. Rather solve it beofre calling this function. 

`snakemake.input.capacity_pv[0]`